liquid-staking/src/delegation.rs

```diff
@@ -0,0 +1,163 @@
+use multiversx_sc::imports::*;
+pub const UNBOND_PERIOD: u64 = 10;
+pub const DELEGATE_ENDPOINT: &str = "delegate";
+pub const UNDELEGATE_ENDPOINT: &str = "unDelegate";
+pub const CLAIM_REWARDS_ENDPOINT: &str = "claimRewards";
```

Comment:
Use `&[u8]` instead of `&str`, you don't need all the overhead of strings. Also, add `b'` in front after you do.

Reply:
Also, unsure why you insist on doing raw calls instead of defining a custom proxy. Example here:
https://github.com/multiversx/mx-ibc-rs/blob/some-tests/client-impls/local-host/src/client_logic.rs#L10-L24

Way less error-prone if you do it that way.